source/_integrations/refoss.markdown

```diff
@@ -0,0 +1,64 @@
+---
+title: Refoss
+description: Integrate Refoss devices
+ha_category:
+  - Switch
+ha_release: 2023.11
+ha_codeowners:
+  - '@ashionky'
+ha_iot_class: Local Polling
+ha_domain: refoss
+featured: true
+ha_config_flow: true
+ha_platforms:
+  - switch
+ha_integration_type: integration
+---
+
+Integrate Refoss devices into Home Assistant.
+
+## Prerequisites
+
+The device needs to be connected to the local network first. There are two ways to connect devices to the local network:
+  - Using the Refoss app (download and install the Refoss app on Google Play) to connect devices to the local network.
+  - Using the Web page to connect devices to the local network.
+    - Computer/mobile phone connected to device factory wifi.
+    - In the browser, access the address 10.10.10.1, then select the local wifi that needs to be added for device configuration.
+
+{% include integrations/config_flow.md %}
+
+### Discover device
+
+  - After integrating Refoss, will start socket broadcast,Home Assistant will automatically discover Refoss devices under the current LAN.
+  
+## Entity naming
+
+The integration uses the following strategy to name its entities:
+
+-  The entity name will be generated by `Device Name` and `Channel Number`.
+
+Single channel device.
+
+Examples:
+
+| Device Name |Channel Number| Entity Name |              
+| ----------- | -----------|------------|
+| `r10`       | `0`           | r10        |
+
+Multi-channel device.
+  
+Examples:
+
+| Device Name | Channel Number | Entity Name |              
+| ----------- |----------------|----------|
+| `r10`       | `0`            | r10      |
+| `r10`       | `1`            | r10-1    |
+
+## Reset Device
+
+Long-press the button on the device to reset it.
+
+## Limitations
+
+- Before set up, the device is connected to the local network and powered on.
+- The integration will occupy port: 9989.
```

Comment:
```suggestion
- The integration will occupy port: 9989.
```
I'm not sure this is a limitation. Could we add this to the prerequisites, where the other networking information is listed?